scripts/install_foxx.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
 
-# -e has been removed so that if an error occurs the PASSWORD File is deleted and not left lying around
-# -u has been removed because we have no guarantees that the env variables are defined
-set -f -o pipefail
+# History
+#
+# -e has been added back, password file deletion should be handled by another 
```

Comment:
I like having all of this context.
I see that this line determines where the password is held: https://github.com/ORNL/DataFed/pull/1187/files#diff-f133cbf9159161dad3622c19800bfb3eda6af5418d0770c42f99288a23a3fe92R214

Is there an easy way to communicate that back to CI and make the pipeline act on this conditionally only if there is a failure?
Or maybe we can make another script that runs cleanup regardless.

Reply:
@t-ramz I'm open to having a discussion about how this should be implemented, however, I don't want to block merging this. Even if passwords are written to a file this only becomes a problem if the script crashes.

Reply:
No worries I just wanted to have it on the docket and in the open!